packages/core/admin/admin/src/content-manager/components/Inputs/index.js

```diff
@@ -266,6 +275,15 @@ function Inputs({
       // in case the default value of the boolean is null, attribute.default doesn't exist
       isNullable={inputType === 'bool' && [null, undefined].includes(fieldSchema.default)}
       description={description ? { id: description, defaultMessage: description } : null}
+      minMaxDescription={
+        minMaxDescription
+          ? {
+              id: minMaxDescription.id,
+              defaultMessage: minMaxDescription.defaultMessage,
+              values: minMaxDescription.values,
+            }
+          : null
+      }
```

Comment:
Why would you want to set this up here and not where the GenericInput sets-up the other labels? https://github.com/strapi/strapi/blob/5e1fda7e13b226ddd2f27526514e0cc32684f36a/packages/core/helper-plugin/lib/src/components/GenericInput/index.js#L110-L129

I think I'd prefer it there instead of passing it down. It also allows to to assign it to the input fields which should support it rather than having these arrays with types you need to check against and you can do special things (as for number) only on for the affected component.